lib/puppet/provider/apt_key/apt_key.rb

```diff
@@ -100,34 +96,34 @@ def self.key_line_hash(pub_line, fpr_line)
 
     fingerprint = fpr_split.last
     return_hash = {
-      :key_fingerprint => fingerprint,
-      :key_long        => fingerprint[-16..-1], # last 16 characters of fingerprint
-      :key_short       => fingerprint[-8..-1], # last 8 characters of fingerprint
-      :key_size        => pub_split[2],
-      :key_type        => nil,
-      :key_created     => Time.at(pub_split[5].to_i),
-      :key_expiry      => pub_split[6].empty? ? nil : Time.at(pub_split[6].to_i),
+      key_fingerprint: fingerprint,
+      key_long: fingerprint[-16..-1], # last 16 characters of fingerprint
+      key_short: fingerprint[-8..-1], # last 8 characters of fingerprint
+      key_size: pub_split[2],
+      key_type: nil,
+      key_created: Time.at(pub_split[5].to_i),
+      key_expiry: pub_split[6].empty? ? nil : Time.at(pub_split[6].to_i),
     }
 
     # set key type based on types defined in /usr/share/doc/gnupg/DETAILS.gz
     case pub_split[3]
-    when "1"
+    when '1'
       return_hash[:key_type] = :rsa
-    when "17"
+    when '17'
       return_hash[:key_type] = :dsa
-    when "18"
+    when '18'
       return_hash[:key_type] = :ecc
-    when "19"
+    when '19'
       return_hash[:key_type] = :ecdsa
     end
 
-    return return_hash
+    return_hash
   end
 
   def source_to_file(value)
-    parsedValue = URI::parse(value)
-    if parsedValue.scheme.nil?
-      fail("The file #{value} does not exist") unless File.exists?(value)
+    parsed_value = URI.parse(value)
+    if parsed_value.scheme.nil?
+      raise("The file #{value} does not exist") unless File.exist?(value)
```

Comment:
the s/fail/raise/ is a false positive I think, because `fail()` is part of Puppet::Util:Logging

This might need "fixing" by adjusting the rubocop config in pdk-module-template

Reply:
I spoke with @DavidS about this, this is not an issue for now.